src/main/java/service/ApiServiceImpl.java

```diff
@@ -0,0 +1,244 @@
+            response.append(inputLine);
+        }
+        in.close();
+
+        return response.toString();
+    }
+
+    public List<Participant> getParticipantById(Long id){
+        ParticipantResponse participantResponse;
+
+        try{
+            participantResponse = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + "participant" + "/" + id,"GET",null),ParticipantResponse.class);
+        }catch (Exception e){
+            throw new CavabungaClientException(e.getMessage());
+        }
+
+        return participantResponse.getData();
+    }
+
+    public List<Participant> getParticipantByUserName(String userName){
+        ParticipantResponse participantResponse;
+
+        try{
+            participantResponse = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + "participant" + "/" + userName,"GET", null),ParticipantResponse.class);
+        }catch (Exception e){
+            throw new CavabungaClientException(e.getMessage());
+        }
+
+        return participantResponse.getData();
+    }
+
+    public List<Participant> getAllParticipants(){
+        ParticipantResponse participantResponse;
+
+        try{
+            participantResponse = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + "participant" + "/","GET",null) , ParticipantResponse.class);
+        }catch (Exception e){
+            throw new CavabungaClientException(e.getMessage());
+        }
+
+        return participantResponse.getData();
+    }
+
+    public void deleteParticipantById(Long id){
+        Response response;
+
+        try {
+            response = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + "participant" + "/" + id, "DELETE", null), Response.class);
+        }catch (Exception e){
+            throw new CavabungaClientException(e.getMessage());
+        }
+    }
+
+    public void saveParticipant(Participant participant){
+        Response response;
+
+        try{
+            if(participant.getId() == null){
+                response = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + participant, "POST", mapper.writeValueAsString(participant)), Response.class);
+            }else{
+                response = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + participant, "PUT", mapper.writeValueAsString(participant)), Response.class);
+            }
+        }catch (Exception e){
+            throw new CavabungaClientException(e.getMessage());
+        }
+    }
+
+    public List<Component> getComponentById(Long id){
+        ComponentResponse componentResponse;
+
+        try {
+            componentResponse = mapper.readValue(this.getApiRequestResult(this.cavabungaURL + ":" + this.cavabungaPort + "/" + "component" + "/" + id, "GET", null), ComponentResponse.class);
```

Comment:
https://stackoverflow.com/questions/3730002/code-line-wrapping-how-to-handle-long-lines